lang/en.json

```
@@ -1647,6 +1647,13 @@
 "SETTINGS.5eAutoCollapseCardN": "Collapse Item Cards in Chat",
 "SETTINGS.5eAutoSpellTemplateL": "When a spell is cast, defaults to begin the process to create the corresponding Measured Template if any (requires TRUSTED or higher player role)",
 "SETTINGS.5eAutoSpellTemplateN": "Always place Spell Template",
+"SETTINGS.5eAttackRollCheckVisibility": {
+  "Name": "Show attack roll result to players",
+  "Hint": "Control visibility of AC check cards.",
+  "GM": "Show only to GM",
+  "Players": "Show to players",
+  "PlayersHideAC": "Show to players, but hide AC"
```

Tweaking the wording of these to match the challenge visibility setting more.

```suggestion
  "All": "Show results & target ACs",
  "HideAC": "Show only results",
  "None": "Hide all"
```